### DISCUSSION TITLE
[SOLVED] I broke my AGS bar by clicking on something in the top right menu

### DISCUSSION THREAD

**Discussion by chipsogdipp** [end-4/dots-hyprland#705] [General]
Disclaimer: I am very new and inexperienced when it comes to most things in the Linux ecosystem, so if the solution is very obvious then I apologize.

Now, I believe I clicked on "tune" and my AGS bar has been like this ever since
![xdph_screenshot_62bc2101](https://github.com/user-attachments/assets/e0b881ba-68be-41f9-8de8-61d62398ce62)
![xdph_screenshot_5c9c6fff](https://github.com/user-attachments/assets/51984cce-ba90-4ede-9a41-d0cbb2335f46)

I have done a couple of searches but haven't found anything helpful, tried reading the wiki's for both end4 and AGS but as far as I was able to see there wasn't any troubleshooting tips for situations like mine

**Comment by end-4** [comment]
select wallpaper again maybe

**Reply by chipsogdipp** [reply]
Didn't work, also tried reinstalling AGS but that didn't work either. Rebooting computer also doesn't work

**Reply by chipsogdipp** [reply]
Oh and every menu is like this, as if the GTK isn't applied or something. From what I could tell the tune button seems to be some sort of live GTK debugger? (Could be completely wrong here), if it is can I reapply GTK themes somehow?

**Reply by chipsogdipp** [reply]
Did a full reinstall of my system, it's working now lol

**Reply by TallenPeli** [reply]
For anyone else who has this problem, try opening Gradience so that it makes a configuration folder. This will prevent the theme from not being applied to ags
